src/BuildingBlocks/DynamicEntity/DynamicEntity/Extensions/StringExtensions.cs

```
@@ -0,0 +1,26 @@
+﻿namespace DynamicEntity.Extensions
+{
+    public static class StringExtensions
+    {
+        public static string ToCamelCase(this string item)
+        {
+            var firstPartUpper = char.ToUpper(item[0]);
+            var secondPartLower = item.Substring(1).ToLower();
+            return $"{firstPartUpper}{secondPartLower}";
+        }
+
+        public static int GetValidIntProperty(this string item)
+        {
+            if (item == null) return 0;
+            int.TryParse(item, out var value);
+            return value;
```

I don't know what is better: to return default(int) value if we can't get valid int property from string or we should to throw an exception?

---

I removed those extensions as vestigial ones. I think that their implementation is not needed yet